### DISCUSSION TITLE
Estimate the total number of Assets/Ops from Dagster OSS

### DISCUSSION THREAD

**Discussion by cody-scott** [dagster-io/dagster#25367] [Q&A] [answered]
Is there a straightforward way to estimate the total number of assets/ops run in the open source dagster instance?
To consider and estimate the total cost of something like dagster+, it would be beneficial to see how our current usage is using our postgres run history.

I'm guessing its a combo of the tables `public.runs` and `public.event_logs` (with some filters applied)?

**Comment by garethbrickman** [comment, marked as answer]
Hey Cody, please [reach out to the Dagster+ Sales team](https://dagster.io/contact)- they'll be able to help you estimate that
